test/DebugInfo/DebugInfo-GV-with-DIE.spt

```diff
@@ -0,0 +1,39 @@
+;; Ensure that reverse translation can handle an Expression in
+;; a DebugGlobalVariable's Variable field.
+;; This is used to preserve a DIExpressions in a DIGlobalVariableExpression.
+
+; RUN: llvm-spirv -to-binary %s -o %t.spv
```

Comment:
To clarify, why the test not matching llvm_14 test?

Reply:
llvm_release_160 does not have the backports:

```
commit 9e112de55989c6d7ee09825446a47cf76288d9e2
Author: LU-JOHN <111294400+LU-JOHN@users.noreply.github.com>
Date:   Fri Jun 9 04:23:38 2023 -0500

    Only generate BuildIdentifier if non-semantic debug is enabled (#2040)
--
commit 7ebb38937029bf02fd03282e15e6652c40260922
Author: LU-JOHN <111294400+LU-JOHN@users.noreply.github.com>
Date:   Fri May 19 15:05:37 2023 -0500

    [DebugInfo] Support translation of DebugBuildIdentifier/DebugStoragePath instruction (#1977)
```
llvm_release_140, llvm_release_150, and llvm_release_170 do have these backports.

llvm_release_160 should be updated to have these backports and then the same testcase can be used for all branches.